packages/utils/telemetry-utils/src/test/mockLogger.spec.ts

```diff
@@ -22,37 +22,37 @@ describe("MockLogger", () => {
 		});
 
 		it("One logged, none expected", () => {
-			mockLogger.sendTelemetryEvent({ eventName: "A", a: 1 });
+			mockLogger.toTelemetryLogger().sendTelemetryEvent({ eventName: "A", a: 1 });
```

Comment:
Once MockLogger is truly internal, the idea would be to make it so this call is unnecessary, I hope?

Reply:
i'm not sure. this is pretty much testing how a mock logger behaves under a telemetry logger, which was what the tests always did, just now it is composition rather than inheritance.